aggregator/src/multicast.rs

```diff
@@ -4,21 +4,33 @@
 //! We expose the `Multicast` trait, mimicking the interface of `aleph_bft::ReliableMulticast`
 
 use std::{
-    fmt::{Debug, Display},
+    fmt::{Debug, Display, Formatter},
     hash::Hash as StdHash,
 };
 
-use aleph_bft::{Signable, SignatureSet};
-use aleph_bft_rmc::ReliableMulticast;
+use aleph_bft_rmc::{MultiKeychain, ReliableMulticast, Signable, Signature};
 use codec::{Codec, Decode, Encode};
 
-use crate::crypto::{Keychain, Signature};
-
 /// A convenience trait for gathering all of the desired hash characteristics.
 pub trait Hash: AsRef<[u8]> + StdHash + Eq + Clone + Codec + Debug + Display + Send + Sync {}
 
 impl<T: AsRef<[u8]> + StdHash + Eq + Clone + Codec + Debug + Display + Send + Sync> Hash for T {}
 
+#[derive(StdHash, PartialEq, Eq, Clone, Copy, Encode, Decode, Debug)]
+pub struct MockHash(pub [u8; 32]);
```

Comment:
Hide this thing under a `cfg(test)` plx, perhaps in a separate file even.

Reply:
moved to tests